tests/runner.py

```diff
@@ -521,20 +556,6 @@ def run_generated_code(self, engine, filename, args=[], check_timeout=True, outp
     assert 'strict warning:' not in ret, 'We should pass all strict mode checks: ' + ret
     return ret
 
-  def build_native(self, filename, args=[]):
```

Comment:
hmm i thought these were used in the benchmark suite, but looks like they are not...